### PR TITLE
Polish v1 items

### PR DESCRIPTION
**Pull request by asl677**

### CODE REVIEW

File: apps/company-landing/app/sections/EmailSection.tsx [hunk 1]
```diff
@@ -9,14 +9,14 @@ import { SuperDisplay } from "design-system/typography";
 
 export const EmailSection = () => {
   const content = {
-    email: "hello@moosejs.dev",
```
Comment (callicles) [inline apps/company-landing/app/sections/EmailSection.tsx]: Please revert this to the old version

File: apps/framework-landing/app/components.tsx/CTAs.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import { ReactNode } from "react";
```
Comment (callicles) [inline apps/framework-landing/app/components.tsx/CTAs.tsx]: To delete if not used

File: apps/framework-landing/app/components.tsx/PlaceholderImage.tsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { cn } from "design-system/utils";
```
Comment (callicles) [inline apps/framework-landing/app/components.tsx/PlaceholderImage.tsx]: Not sure if this is needed either

File: apps/framework-landing/app/sections/EmailSection.tsx [hunk 4]
```diff
@@ -9,14 +9,14 @@ import { SuperDisplay } from "design-system/typography";
 
 export const EmailSection = () => {
   const content = {
-    email: "hello@moosejs.dev",
+    email: "hello@moosejs.com",
```
Comment (callicles) [inline apps/framework-landing/app/sections/EmailSection.tsx]: Revert this

**Review by callicles — CHANGES_REQUESTED** [review]

**Review by callicles — APPROVED** [review]

**Comment by asl677** [comment]
Some reference issue with TrackCtaButton on community page though it works its still throwing an error.

**Comment by asl677** [comment]
yaml might need love from @callicles

**Pull request merged**